src/components/subscription/ExtendSubscriptionModal.tsx

```
@@ -0,0 +1,174 @@
+              {[1, 3, 6, 12].map((months) => {
+                const hasDiscount = months === 12 || months === 6 || months === 3;
+                return (
+                  <button
+                    key={months}
+                    onClick={() => setSelectedMonths(months)}
+                    className={`relative px-4 py-3 rounded-lg border transition-all duration-200 font-light ${
+                      selectedMonths === months
+                        ? "border-blue-500/50 bg-blue-500/20 text-blue-300"
+                        : "border-white/20 bg-white/5 text-white/70 hover:border-white/30 hover:bg-white/10"
+                    }`}
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)